chocolatey-community/chocolatey-au
# (#32) Add a build pipeline

## Description Of Changes

- Add a Team City build pipeline so that the module can be built automatically.
- Add a GitHub Actions build pipeline for visibility into the automated builds.
- Rename module to `Chocolatey-AU` and the corresponding Chocolatey package to `chocolatey-au`.

## Motivation and Context

- Make it more reliable to have a build of Chocolatey-AU
- Allow the module to be released to PowerShell Gallery and Chocolatey Community Repository.

## Testing

- Run `./build.ps1` locally ensure it works.
- Run the build on dev Team City server.
- [Run the GitHub Action against my personal fork.](https://github.com/corbob/chocolatey-au/actions/workflows/build-module.yml)

### Operating Systems Testing

- Windows 10 22H2 (Locally)
- Team City agent
- GitHub Action agent

## Change Types Made


## Change Checklist


## Related Issue

* Fixes #32 
* ENGTASKS-3526
